examples/image.md

```diff
@@ -0,0 +1,234 @@
+
+```cpp
+void SummaryWriter<Filewriter>::Image(const std::string& tag,
+                                      int step,
+                                      arma::Mat<eT>& matrix,
+                                      mlpack::data::ImageInfo& info,
+                                      mlboard::Filewriter& fw,
+                                      const std::string& displayName,
+                                      const std::string& description)
+```
+
+The API accepts `tag`, `step`, `encodedImage` (An `arma::mat` which has image data), `mlpack::data::ImageInfo` object, `mlboard::Filewriter` object, name and description.
+
+Following is a snippet that would log a multiple image which is stored in `arma::mat` for 1 step in temp directory.
+
+```cpp
+#include <mlboard/mlboard.hpp>
+#include <iostream>
+#include <chrono> 
+#include <ctime> 
+#include <future>
+
+int main()
+{
+  std::chrono::time_point<std::chrono::system_clock> start, end; 
+  start = std::chrono::system_clock::now(); 
+  mlboard::FileWriter f1("temp");
+  
+  // Following line are just there to come up with a arma::mat of images.
+  // If you already have a matrix with images you can avoid the following lines.
+  arma::Mat<unsigned char> matrix;
+  mlpack::data::ImageInfo info;
+  std::vector<std::string> files = {"./assets/single_image.jpg",
+        "./assets/single_image.jpg"};
+
+  // Creating the matrix which has image.
+  mlpack::data::Load(files, matrix, info, false);
+
+  // Now we can log the matrix.
+  mlboard::SummaryWriter<mlboard::FileWriter>::Image(
+        "Multiple Image", 1, matrix, info, f1, "Sample Multiple Image",
+        "This is a Sample image logged using mlboard.");
+  f1.Close();
+  end = std::chrono::system_clock::now();
+  std::chrono::duration<double> elapsed_seconds = end - start;
+  std::time_t end_time = std::chrono::system_clock::to_time_t(end); 
+  
+  std::cout << "finished computation at " << std::ctime(&end_time) 
+            << "elapsed time: " << elapsed_seconds.count() << "s\n";
+}
+```
+
+The output would be similar to [Log multiple image](#2-multiple-image)
+
+Note : This api takes quite a time to log the image since it first have to process the image stored in `arma::mat`. The way it process is, it saves the image using `mlpack::data::Load()` api for image and then reads it back in encoded format. It is something similar to what mxnet logging tool does [here](https://github.com/TeamHG-Memex/tensorboard_logger/blob/0ec6f1147b7cea8c4b68d26637781ccd8ec036b9/tensorboard_logger/tensorboard_logger.py#L157). Hence it is recomended to log image summary using a `arma::mat` in a seperate thread to avoid increase in training time. Also make sure that all the images are of same height and width,since mlapck::load() function only supports loading images of same sizes.
```

Comment:
No this is the right place, It is possible to log two different size images using tensorbaord, but that is not possible using arma mat, because matrix cannot be of odd sizes like one image having 15 pixels and other images have 17 pixels.

If you read two images as string then it is possible to log both the images irresspective of their sizes